crates/rust/src/lib.rs

```diff
@@ -1218,8 +1248,18 @@ impl WorldGenerator for RustWasm {
         types: &[(&str, TypeId)],
         _files: &mut Files,
     ) {
+        let mut to_define = Vec::new();
+        for (name, ty_id) in types {
+            let full_name = full_wit_type_name(resolve, *ty_id);
+            if let Some(TypeGeneration::Remap(_)) = self.with.get(&full_name) {
```

Comment:
(same as above)

Reply:
Sure. Done in https://github.com/bytecodealliance/wit-bindgen/pull/1173/commits/613d2fd95f360e554613d435e74c1c52b85617bf